endpoints/api/test/test_security.py

```diff
@@ -5767,8 +5767,9 @@ def test_all_apis_tested(app):
     for rule in app.url_map.iter_rules():
         endpoint_method = app.view_functions[rule.endpoint]
 
-        # Verify that we have a view class for this API method.
-        if not "view_class" in dir(endpoint_method):
+        # Verify that we have a view class for this API method,
+        # and that we're not testing a v2 endpoint.
+        if not "view_class" in dir(endpoint_method) or "v2" in rule.endpoint:
```

Comment:
these tests fail if any views other than the ones in `endpoints/api` are classes rather than functions. I extended the check a little to exclude views from the v2 api, since this test exclusively tests stuff inside `endpoints/api`